docs/SETUP.md

```diff
@@ -118,4 +124,8 @@ For production builds—which minify the assets, remove source maps, etc.—run
 
 ### Testing in Browser
 
-After starting the development server with `npm run dev` or `npm start` and launching the SQE database Docker container, you can access the website at `localhost:9090`.  A default user for testing has already been installed to the database; the user name is `test` and the password is `asdf`.
\ No newline at end of file
+After starting the development server with `npm run dev` or `npm start` and launching the SQE database Docker container (`docker start SQE-Databse`), you can access the website at `localhost:9090`.  A default user for testing has already been installed to the database; the user name is `test` and the password is `asdf`.
+
+### Note
+
+One should shutdown the SQE database Docker after usage with `docker stop SQE-Database`. If not shut down properly, one will run into problems to restart the Docker after logging out. In this case, one may clean up the system using `docker rm $(docker ps -q -f status=exited)`.
```

Comment:
Suggest a slight rewording: `One should shutdown the Docker container after usage with `docker stop SQE-Database`. If not shut down properly, problems may be encountered when attempting to restart the container after shutdown or logging out.`